readthedocs/projects/forms.py

```diff
@@ -241,12 +241,12 @@ def __init__(self, *args, **kwargs):
         )
 
         default_choice = (None, '-' * 9)
-        all_versions = self.instance.versions.values_list(
-            'identifier',
-            'verbose_name',
-        )
+        all_versions_choices = [
+            (v.commit_name, v.verbose_name)
+            for v in self.instance.versions.all()
+        ]
```

Comment:
Would be useful to have a comment here explaining why we changed this logic. I could see someone trying to change it back.